### DISCUSSION TITLE
How to Merge code from Two Different Repository

### DISCUSSION THREAD

**Discussion by 7dcars** [community/community#23741] [New to GitHub]
<p>Dear Team,</p>
<p>We have 02 different Repository and we want to merge the code of both repository into One.</p>
<p>e.g.: Repo X and Repo Y, We wanted to merge into Repo X</p>
<p>Thanks &amp; Regards,<br>
Roni<br>
Head of Operation<br>
ATT<br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="http://attcabs.com/assets/img/favicon_io/favicon.ico" class="site-icon" width="48" height="48">
      <a href="http://www.attcabs.com" target="_blank" rel="nofollow noopener">attcabs.com</a>
  </header>
  <article class="onebox-body">
    <img src="http://attcabs.com/assets/img/favicon_io/favicon.ico" class="thumbnail onebox-avatar" width="48" height="48">

<h3><a href="http://www.attcabs.com" target="_blank" rel="nofollow noopener">Book Outstation Cabs, Local, Oneway Pickup Drop &amp; Airport Transfer</a></h3>

<p>Cabs in Pune, Local Cab in Pune, Oneway Pickup Drop &amp; Airport Transfer</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi Roni! :wave: Welcome to the Community!</p>
<p>It sounds like you’re looking for the <code>--allow-unrelated-histories</code> command!</p>
<p>This third party article explains how to use it:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/19803303/181079163-b09a9a39-ca9c-4ab7-8e25-6382f371c7be.png" class="site-icon" width="32" height="32">
      <a href="https://medium.com/@spences10/git-allow-unrelated-histories-a39a3814b981" target="_blank" title="01:20PM - 30 June 2017">Medium – 30 Jun 17</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/142;"><img src="https://user-images.githubusercontent.com/19803303/181079165-e2b7eba1-68a9-4031-8c19-06447c811323.png" class="thumbnail" width="690" height="142" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/121521c5aa9dc7509abb8952e508411fcc9f85fa_2_690x142.png, https://aws1.discourse-cdn.com/github/original/2X/1/121521c5aa9dc7509abb8952e508411fcc9f85fa.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/121521c5aa9dc7509abb8952e508411fcc9f85fa.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/121521c5aa9dc7509abb8952e508411fcc9f85fa_2_10x10.png"></div>

<h3><a href="https://medium.com/@spences10/git-allow-unrelated-histories-a39a3814b981" target="_blank">Git ` — allow-unrelated-histories`</a></h3>

<p>How to combine two separate unrelated Git repositories into one with single history timeline.</p>

  <p><span class="label1">Reading time: 2 min read</span>
    </p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
